### DISCUSSION TITLE
Need help in calculating total time spent in hours on enrolled and consumed coursees.

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Knowlg/Community#193] [Q&A]
Hello team
Hope you are doing well 

Background -
We have updated few screens for sunbird-Ed and trying to add few more details

Issue Details -
I wanted to know weather we have any API that provide us actual spent time on the courses by specific user. And in which database table we can find this data. We don't want to calculate it based on start date and end as it can not be precise

@komalm, @manishakapadnis, @AmiableAnil, @krgauraw ,@pallakartheekreddy

**Comment by pallakartheekreddy** [comment]
@paritshivani Above functionality is part of Lern building block. Can you create this thread in the below discussion forum.
https://github.com/orgs/Sunbird-Lern/discussions

@AmiableAnil FYI...

**Reply by paritshivani** [reply]
added there. Thank you
